src/tests/EdFi.DataManagementService.Tests.E2E/Features/Properties/Properties.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Validate Extra Properties are being removed
+    Tests that perform POST and PUT requests with extra properties,
+    paired with a GET request that proves the extra properties have been removed
+
+        Background:
+             When a POST request is made to "ed-fi/academicWeeks" with
+                  """
+                  {
+                  "weekIdentifier": "LastWeek",
+                  "schoolReference": {
+                  "schoolId": 255901001,
+                  "link": {
+                  "rel": "School",
+                  "href": "/ed-fi/schools/20ec19e5070245128a30fdcc6925bb09"
+                  }
+                  },
+                  "beginDate": "2024-05-30",
+                  "endDate": "2024-05-30",
+                  "totalInstructionalDays": 0,
+                  "_etag": "5250168731208835753",
+                  "_lastModifiedDate": "2024-05-30T22:30:57.509Z"
+                  }
+                  """
+             Then it should respond with 201
+              And the response headers includes
+                  """
+                    {
```

Comment:
The curly brackets should start under the first double quote from the line above.

Reply:
Fixed